go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/nats-io/nats-server/v2
 
-go 1.15
+go 1.16
 
 require (
-	github.com/klauspost/compress v1.11.7
-	github.com/minio/highwayhash v1.0.0
-	github.com/nats-io/jwt/v2 v2.0.0-20210208203759-ff814ca5f813
+	github.com/klauspost/compress v1.11.12
+	github.com/minio/highwayhash v1.0.1
```

Comment:
This patch version bump seems mostly updates to run in graviton so should be ok: https://github.com/minio/highwayhash/compare/v1.0.0...v1.0.1

Reply:
since I updated jwt, we bumped these as well

Reply:
We decided to update all deps since this is a new release.